### ISSUE TITLE
Staging Review finding: Using a screen reader, unable to select combobox options

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#5127]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** vfs-mhv-secure-messaging
**Product:** mhv-secure-messaging
**Feature:** Oracle Health curated list

## Findings details
**VA.gov Experience Standard - issue:** [User can't complete a necessary flow or hits a dead end.](https://design.va.gov/about/experience-standards/#user-can-complete-necessary-flows)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On Android, with Talkback on, a user is unable to select any options in the "Select a care team" combobox. When swiping from the combobox with results, focus moves to the "x" and the options disappear, and the user can reactivate the results, but cannot swipe to them. 

<details><summary>Link, screenshot or steps to recreate</summary>

https://github.com/user-attachments/assets/55a8f958-7f53-462d-98b2-71833e90182a

</details>

## Recommended action

The combobox will need to be reworked to allow for focus to move to the results. 
Note: this is a design system issue

Some options:
- put the results list in the DOM before the close "x", so that swiping would move from the input to the results list container
- Or, when the user activates the caret, put focus onto the search results container (currently focus resets to the input, and when swiping, focus moves to the "x", closing the results

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by caw310** [comment]
DST considers this an enhancement. We will look into this in a future sprint.
